### PR TITLE
Javadoc fixes

### PR DESCRIPTION
**Pull request by nrgraham23**
Javadoc fixes.  Added missing @params and @throws as well as adding definitions for the 
@throws tags.  Also added suppression statements in areas where it did not make sense to add
an @param for certain variables.

Commenting fixes

Signed-off-by: Nathaniel Graham ngraham@lanl.gov

@hppritcha @jsquyres 

:bot:milestone:v2.0.0
;bot:label:enhancement

### CODE REVIEW

**Comment by lanl-ompi** [comment]
Test FAILed.

**Comment by lanl-ompi** [comment]
Test FAILed.

**Comment by nrgraham23** [comment]
bot:retest

File: ompi/mpi/java/java/CartComm.java [hunk 1]
```diff
@@ -104,7 +104,7 @@ protected CartComm(long[] commRequest)
  * <p>The new communicator can't be used before the operation completes.
  * The request object must be obtained calling {@link #getRequest}.
  * @return copy of this communicator
- * @throws MPIException
+ * @throws MPIException Signals that an MPI exception of some sort has occurred. Signals that an MPI exception of some sort has occurred.
```
Comment (hppritcha) [inline ompi/mpi/java/java/CartComm.java]: this is doubled.

Comment (nrgraham23) [inline ompi/mpi/java/java/CartComm.java]: Doubled?

Comment (hppritcha) [inline ompi/mpi/java/java/CartComm.java]: yep, the sentence is repeated.

Comment (nrgraham23) [inline ompi/mpi/java/java/CartComm.java]: Fixed

**Pull request merged**